src/gui/ignorelisteditor.cpp

```diff
@@ -34,7 +34,7 @@ static const int skippedLinesRole = Qt::UserRole;
 static const int isGlobalRole = Qt::UserRole + 1;
 
 IgnoreListEditor::IgnoreListEditor(QWidget *parent)
-    : QDialog(parent, Qt::Sheet)
+    : QDialog(parent)
```

Comment:
Should we remove `Qt::Sheet` from the other locations as well, now that this is handled differently in that utility method?